packages/suite/src/components/suite/modals/ReduxModal/CardanoWithdrawModal.tsx

```diff
@@ -0,0 +1,156 @@
+    align-items: center;
+    margin-bottom: 5px;
+`;
+
+export const Row = styled.div`
+    display: flex;
+    width: 100%;
+    margin-top: 24px;
+`;
+
+export const Line = styled.div`
+    display: flex;
+    width: 100%;
+`;
+
+export const LeftPadding10 = styled.div`
+    padding-left: 10px;
+`;
+
+export const Column = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    max-width: 100%;
+`;
+
+export const Title = styled.div`
+    display: flex;
+    color: ${({ theme }) => theme.legacy.TYPE_DARK_GREY};
+    ${typography.highlight}
+    align-items: center;
+    margin-bottom: 16px;
+`;
+
+export const Actions = styled.div`
+    display: flex;
+    align-items: center;
+    width: 100%;
+    margin-top: 24px;
+    justify-content: center;
+`;
+
+export const Text = styled.div`
+    color: ${({ theme }) => theme.legacy.TYPE_LIGHT_GREY};
+    margin-bottom: 8px;
+    margin-top: 8px;
+    ${typography.hint}
+`;
+
+export const Content = styled.div`
+    display: flex;
+    overflow: hidden;
+    padding-left: 10px;
+    margin-left: -10px;
+    width: 100%;
+`;
+
+export const Value = styled.div`
+    ${typography.hint}
+    color: ${({ theme }) => theme.legacy.TYPE_DARK_GREY};
+    font-variant-numeric: tabular-nums slashed-zero;
+    width: fit-content;
+    background: ${({ theme }) => theme.legacy.BG_LIGHT_GREY};
+    border: 1px solid ${({ theme }) => theme.legacy.STROKE_GREY};
+    border-radius: ${borders.radii.xs};
+    word-break: break-all;
+    padding: 10px;
+    display: flex;
+`;
+
+type Props = {
+    onCancel: () => void;
+};
+
+export const CardanoWithdrawModal = ({ onCancel }: Props) => {
+    const { voteAbstain, voteDelegate } = useCardanoStaking();
+    const account = useSelector(state => state.wallet.selectedAccount.account);
+    if (!account || account.networkType !== 'cardano') {
+        throw Error('CardanoWithdrawModal used for other network');
+    }
+
+    const cardanoNetwork = account.symbol === 'ada' ? 'mainnet' : 'preview';
+    const { trezorDRep } = useSelector(state => state.wallet.cardanoStaking[cardanoNetwork]);
+    const trezorDRepBech32 = trezorDRep?.drep.bech32;
+
+    return (
+        <NewModal onCancel={onCancel}>
+            <StyledH2>
+                <FormattedMessage
+                    id="TR_CARDANO_WITHDRAW_MODAL_TITLE"
+                    defaultMessage="Delegate voting rights"
+                />
+            </StyledH2>
+            <Row>
+                <FormattedMessage
+                    id="TR_CARDANO_WITHDRAW_MODAL_DESCRIPTION"
+                    defaultMessage="When withdrawing your rewards, you can choose to support the Cardano ecosystem by delegating your community voting rights. Your votes will be delegated to 5Binaries, the operators of Cardano staking in Trezor Suite. This helps strengthen the network's resilience, sustainability, and community-driven governance. If you prefer, you can easily opt out of governance."
```

Comment:
```suggestion
                    defaultMessage="When withdrawing your rewards, you can choose to support the Cardano ecosystem by delegating your community voting rights. Your votes will be delegated to a representative of Five Binaries, the operators of Cardano staking in Trezor Suite. This helps strengthen the network's resilience, sustainability, and community-driven governance. If you prefer, you can easily opt out of governance."
```

Just a small text correction, to match the legal name of the company.